config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```diff
@@ -92,7 +92,9 @@ periodics:
         wget -q -O /usr/local/bin/snyk https://github.com/snyk/snyk/releases/download/v1.605.0/snyk-linux && chmod +x /usr/local/bin/snyk; \
         mkdir -p "${ARTIFACTS}"; \
         [[ -z "${SNYK_TOKEN}" ]] && echo "SNYK_TOKEN env var is not set, required for snyk scan" && exit 1; \
-        snyk test --json-file-output=${ARTIFACTS}/result_all.json || echo "failed to run snyk scan, exit code $?" && exit 1; \
+        echo "$PWD"; \
+        echo "SNYK_TOKEN is present"; \
+        snyk test --json-file-output=${ARTIFACTS}/result_all.json || [[ $? -gt 1 ]] && echo "failed to run snyk scan" && exit 1; \
```

Comment:
> [[ $? -gt 1 ]] && echo "failed to run snyk scan" && exit 1

`snyk test` returns 2 or 3 exit status for failure (1 means vulns founds, see PR desc)